### DISCUSSION TITLE
When sending an Email from SM, can the "Add to Log" select be changed like the Email and Subject can be?

### DISCUSSION THREAD

**Discussion by kernny** [sheltermanager/asm3#1364] [Q&A / Help] [answered]
![Screen Shot 2023-11-19 at 3 10 16 pm](https://github.com/sheltermanager/asm3/assets/91639986/a408b98c-52f6-44ac-b53f-5fc11ff29ccf)

Just looking to see if there is an option to control the log selection for when sending different email types. It seems to Default to the Email - Adoption Email sent. Would love to know if I can control that default setting too.

**Comment by bobintetley** [comment, marked as answer]
It defaults to whichever log type is chosen as the default under Settings->Options->Defaults->Log Type

**Reply by kernny** [reply]
Thanks again!!!
